bot.py

```diff
@@ -53,15 +55,22 @@
 ]
 
 
-def seconds_until_9am():
+def seconds_until_9am() -> float:
     now = datetime.datetime.now()
     target = (now + datetime.timedelta(days=1)).replace(hour=9, minute=0, second=0, microsecond=0)
     diff = (target - now).total_seconds()
     print(f"{target} - {now} = {diff}")
     return diff
 
+condslist = [
+    dt.Condition("Darren is never wrong, respect your old GMs", dt.Contains("darren")),
+    dt.Condition("Talking about _toes_, are we. :eyes:", dt.Contains("feet") | dt.Contains("foot") | dt.Contains("toe")),
+    dt.Condition("Please refer to Tom by his proper title, Supreme High Guildmaster Tom", dt.Contains("Tom") & Not(dt.Contains("Supreme High Guildmaster Tom"))),
```

Comment:
Since we call `.lower()` on content the bot sees. It will never meet the condition for "Supreme High Guildmaster Tom" ?

Either we handle the `.lower()` in the logic code internally so this doesn't matter, or we conform to lowercase conditions.

Reply:
oh yeah oops forgot that rip me